lib/cdo/crowdin/project.rb

```diff
@@ -53,7 +53,9 @@ def export_file(file, language, etag: nil, attempts: 3, only_head: false)
       end
 
       only_head ? self.class.head("/export-file", options) : self.class.get("/export-file", options)
-    rescue Net::ReadTimeout => error
+
+      timeouts = [Net::ReadTimeout, Net::OpenTimeout]
+    rescue *timeouts => error
```

Comment:
nit: I'd prefer something like

```suggestion
    rescue Net::ReadTimeout, Net::OpenTimeout => error
```

Reply:
ohhh that's how you do it 😅 I initially tried `rescue [Net::ReadTimeout, Net::OpenTimeout] => error` and was getting blocked by the linter. I thought it was the inline array, didn't realize I could just leave out the brackets!